node/src/BaseClient.ts

```diff
@@ -2292,6 +2293,42 @@ export class BaseClient {
         return this.createWritePromise(createSRem(key, members));
     }
 
+    /**
+     * Iterates incrementally over a set.
+     *
+     * See https://valkey.io/commands/sscan for details.
+     *
+     * @param key - The key of the set.
+     * @param cursor - The cursor that points to the next iteration of results. A value of `"0"` indicates the start of the search.
+     * @param options - The {@link BaseScanOptions}.
+     * @returns An array of the cursor and the subset of the set held by `key`. The first element is always the `cursor` and for the next iteration of results.
+     * `"0"` will be the `cursor` returned on the last iteration of the set. The second element is always an array of the subset of the set held in `key`.
+     *
+     * @example
+     * ```typescript
+     * // Assume key contains a set with 200 members
+     * let cursor = "0";
+     * let result = [][];
+     * do {
+     *   result = await client.sscan(key1, cursor, {
+     *          count: 20,
+     *      });
+     *   cursor = result[0];
+     *   const stringResults: string[] = result[1];
+     *
+     *   console.log("SSCAN iteration:");
+     *   stringResults.forEach(i => i + ", ");
+     * } while (!cursor.equals("0"));
```

Comment:
Can you also add a sample output? Ensure that this block works.

Reply:
The print statement does not align with the output below. Make sure that the example run correctly as users should be able to copy paste the important bits in their implementations.